src/redux/reducers/componentConfig/plotDataLoading.js

```diff
@@ -0,0 +1,16 @@
+import { initialPlotDataState } from './initialState';
+
+const plotDataLoading = (state, action) => {
+  const { plotUuid } = action.payload;
+  return {
+    ...state,
+    [plotUuid]: {
+      ...initialPlotDataState,
```

Comment:
Check if we still need this

Reply:
not needed anymore as it is initially initialized when loading the plot config, so this is removed.